### PR TITLE
blog_forms PR

### PR DESCRIPTION
**Pull request by kkasaija**
- Created a method called  `current_user`
- Created forms to Create `posts` and `comments`
- Created means to add likes to `posts`

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def current_user
+    User.first
+  end
```
Comment (FrederickMih) [inline app/controllers/application_controller.rb:4]: 👍

File: config/routes.rb [hunk 2]
```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   root "users#index"
   resources :users, only: [:index, :show] do 
-    resources :posts,only: [:index, :show]
+    resources :posts,only: [:index, :show, :new, :create] do
+      resources :comments, only:[:new, :create] 
+      resources :likes, only:[ :create]
```
Comment (FrederickMih) [inline config/routes.rb:6]: 👍

File: config/routes.rb [hunk 3]
```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   root "users#index"
   resources :users, only: [:index, :show] do 
-    resources :posts,only: [:index, :show]
+    resources :posts,only: [:index, :show, :new, :create] do
```
Comment (FrederickMih) [inline config/routes.rb:4]: - [Optional] @Kasaija-Kenneth, in this milestone, we just limit ourselves to `new` and `create` routes. Keep up 🚀 

Your work is amazing 👍

**Review by FrederickMih — APPROVED** [review]
## Hi 🙋 @Kasaija-Kenneth, 

You did a fantastic job on this project  🎉 🎉 🎉 🍾 🍾 🍾 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉

## Highlights </img>


- Descriptive README.md file ✔️ 
- All requested forms added ✔️ 
- PR is professional ✔️ 
- Linters checks passed ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 3 above)
Comment (kkasaija) [reply config/routes.rb:4]: Thank you so much @FrederickMih. Your advice is noted

**Pull request merged**